coding/python_style_guide.rst

```diff
@@ -553,6 +553,21 @@ When tempted to use ``l``, use ``L`` instead.
 
   This matches the `PEP 8 standard <https://www.python.org/dev/peps/pep-0008/#names-to-avoid>`_ but is repeated here for emphasis.
 
+.. _style-guide-py-naming-metaclasses:
+
+Always use ``cls`` for the first argument to metaclass instance methods
+-----------------------------------------------------------------------
+
+For regular classes ``self`` is used, but for class methods and hence also for metaclass instance
+methods, ``cls`` should be used instead.
+
+Also add ``# noqa N805`` to the relevant line in order to support older versions of flake8.
```

Comment:
"older" begs the question of "than what?"...

If a fixed flake8 is about to hit, should we wait, update the stack, and then not need this line at all?

Reply:
We need PyCQA/pep8-naming#47 to end up in a release of pep8-naming so it might not be a long wait.